torch/csrc/jit/codegen/cuda/utils.h

```diff
@@ -7,6 +7,8 @@ namespace jit {
 namespace fuser {
 namespace cuda {
 
+enum class KernelIndexMode { INT32, INT64 };
```

Comment:
nit: Should this be placed in type.h?

Reply:
Moved to type.h thanks! Originally thought that it was not part of the IR, but I guess type.h also makes sense.